### ISSUE TITLE
public identifiers syntax highlighted

### ISSUE THREAD

**Issue by leemorgan-mobile** [nim-lang/nim-mode#51]
ie things with a trailing '*' are highlighted differently

**Comment by reactormonk** [comment]
Could you rephrase? I'm not entirely sure I understand what you intend to say.

**Comment by lcm337** [comment]
Sure. So the proc name has a different face if it is exported/public.

(Or exported type or variable etc)

It would be nice to look at a module and easily see what's exported.

Thanks!

**Comment by reactormonk** [comment]
Isn't that usually what a docgen does for you?
